README.md

```diff
@@ -158,6 +158,21 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system automatically updates the dynamic configuration. It is
 enabled by default.
 
+```yaml
+    rhc_insights:
+      ansible_host: "example-host"
+```
+
+Configures the ansible host name with a custom value for the system record
+in Host Based Inventory (HBI). This host name is used in playbooks by remediations.
+It is `null` by default and will use the system host name if not specified.
+
+Possible values of this variable:
+
+* `null` or an empty value: the ansible host name is not changed.
```

Comment:
What is an empty value? An empty string? The `omit` 
magic value?

```suggestion
* `null` or an empty value: the Ansible host name is not changed.
```

Reply:
like this:
```yaml
    rhc_insights:
      ansible_host:
```
This is an "empty" value in YAML which is translated to a `null` - see https://yaml.org/spec/1.2.2/#24-tags "Example 2.21 Miscellaneous"

I say "empty" because, although it looks empty, that there is nothing there, in YAML this means to assign the value `null`

Reply:
I see. I knew about this feature of YAML, it just didn’t come to my mind.

Since the so-called empty value translates to _null_ on the YAML level, it may not be worth mentioning here: the value is still _null_ and the variations of the syntax are property of the YAML format, not Ansible or the role.

Reply:
I found out that an empty string value yields inconsistent results:

- `ansible_host=` is put into the config file, causing the value not get updated on upload.
- The Client is called with `--ansible_host=`, removing the value from the Inventory.

Even though this “empty value” refers to the YAML syntax `value:`, it sounds ambiguous to me. I’d rather not describe features of YAML and be more explicit with empty strings.

```suggestion
* `null`: the ansible host name is not changed.
```

if we wanted to use an empty string to reset the value, which I don’t prefer, or

```suggestion
* `null` or an empty string: the ansible host name is not changed.
```

if we chose to consider an empty string an empty value, equivalent to `null` or `omit`.

It may be worth mentioning `omit` too.

```suggestion
* `null`, `omit`, or an empty string: the ansible host name is not changed.
```